.github/workflows/validate-bicep.yaml

```diff
@@ -33,37 +33,14 @@ jobs:
     steps:
       - name: Check out repo
         uses: actions/checkout@v3
-      
-      - name: Clone docs repo
-        uses: actions/checkout@v3
-        with:
-          token: ${{ secrets.GH_RAD_CI_BOT_PAT }}
-          repository: radius-project/docs
-          path: temp/radius-project/docs
-          ref: edge
-      - name: Clone samples repo
```

Comment:
i'm glad this is being removed

Reply:
Yeah, cross-repo depdencies are really hard to use. This flew under the radar because the bicep validation job is optional.